modules/gitops-deleting-rolloutmanager-custom-resource.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/argo-rollouts-for-progressive-delivery-of-deployments.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-deleting-rolloutmanager-custom-resource_{context}"]
+= Deleting RolloutManager custom resource
+
+Uninstallation of {gitops-title} Operator does not remove any resources it created. To uninstall the progressive delivery of deployments by using {gitops-title}, you must explicitly delete the resources by deleting the RolloutManager custom resource (CR) and then uninstall the {gitops-title} Operator.
+
+.Prerequisites
+
+* {gitops-title} 1.9.0, or a later version, is installed in your cluster.
+* RolloutManager custom resources are available in the namespace of your choice.
+
+.Procedure 
+
+. Log in to the {product-title} web console. 
```

Comment:
Again, probably need to mention they need cluster admin permissions and need to go to the Administrator perspective